news/3605.bugfix.rst

```diff
@@ -0,0 +1 @@
+Specify an encoding of UTF-8 when opening text files.
```

Comment:
Even when rewording after the changes I'm requesting, this should state that we basically open *all* text files using  utf-8 *explicitly* and do so for both reading and writing

Reply:
I agree, but that also depends on how we'll handle the opening of files in the hooks. I'll update the comment for now.